Samples/SimpleAtomics/Program.cs

```diff
@@ -58,35 +58,32 @@ class Program
         static void Main()
         {
             // Create main context
-            using (var context = Context.CreateDefault())
+            using var context = Context.CreateDefault();
+
+            // For each available device...
+            foreach (var device in context)
             {
-                // For each available device...
-                foreach (var device in context)
-                {
-                    // Create accelerator for the given device
-                    using (var accelerator = device.CreateAccelerator(context))
-                    {
-                        Console.WriteLine($"Performing operations on {accelerator}");
-                        var kernel = accelerator.LoadAutoGroupedStreamKernel<
-                            Index1D, ArrayView<int>, int>(AtomicOperationKernel);
-                        using (var buffer = accelerator.Allocate1D<int>(7))
-                        {
-                            // Initialize buffer to zero
-                            buffer.MemSetToZero();
+                // Create accelerator for the given device
+                using var accelerator = device.CreateAccelerator(context);
+                Console.WriteLine($"Performing operations on {accelerator}");
+
+                var kernel = accelerator.LoadAutoGroupedStreamKernel<
+                    Index1D, ArrayView<int>, int>(AtomicOperationKernel);
+
+                // Initialize buffer to zero
+                using var buffer = accelerator.Allocate1D<int>(7);
+                buffer.MemSetToZero();
 
-                            // Launch buffer.Length many threads and pass a view to buffer
-                            kernel(1024, buffer.View, 4);
+                // Launch buffer.Length many threads and pass a view to buffer
+                kernel(1024, buffer.View, 4);
 
-                            // Wait for the kernel to finish...
-                            accelerator.Synchronize();
+                // Wait for the kernel to finish...
+                accelerator.Synchronize();
```

Comment:
nit: same as above; However, what happens under the hood could be described here.